group_vars/mash_servers

```diff
@@ -1766,7 +1768,33 @@ mongodb_managed_databases_auto: |
 #                                                                      #
 ########################################################################
 
+########################################################################
+#                                                                      #
+# mosquitto                                                           #
+#                                                                      #
+########################################################################
+
+mosquitto_enabled: false
+
+mosquitto_identifier: "{{ mash_playbook_service_identifier_prefix }}mosquitto"
+
+mosquitto_base_path: "{{ mash_playbook_base_path }}/{{ mash_playbook_service_base_directory_name_prefix }}mosquitto"
 
+mosquitto_uid: "{{ mash_playbook_uid }}"
+mosquitto_gid: "{{ mash_playbook_gid }}"
+
+mosquitto_systemd_required_services_list: |
+  {{
+    (['docker.service'])
+  }}
```

Comment:
This seems to match the default value from the role's `defaults/main.yml`, so it can be removed from here.